roles/check_dependencies/tasks/main.yml

```diff
@@ -81,6 +81,15 @@
   set_fact:
     internal_have_locate: "{{ internal_have_locate_cmd.rc == 0 }}"
 
+- name: gather internal_have_systemctl
+  raw: command -v systemctl
```

Comment:
Cool, but makes me wonder if we are missing a similar check for `chkconfig` that should have been used in some of the JBoss product roles? 

^ @noahl

Reply:
I agree that we need `chkconfig`. I don't see it in my copy of check_dependencies, so I think it's likely missing.

Reply:
Do we want to add that as part of this PR?